src/check-update.ts

```diff
@@ -0,0 +1,250 @@
+import { BUILDER_TAG, HOOKS_TAG, RUNTIME_TAG } from "./mod.ts";
```

Comment:
I noted that these were in a `deps.ts` file, but I pulled in a commit and it upended that. Shall I create that file again?